demokratie-live/democracy-client
# Tutorial Screen -Design

**Description:**

Implemented Tutorial Screen

**How to test:**

- Start App on diferent Devices - see how it looks (requires reinstall or Menu->Tutorial)
- Check if it compiles on iOS
